### ISSUE TITLE
ARIA live region for page changes

### ISSUE THREAD

**Issue by dwhieb** [digitallinguistics/app#16]
When the page changes, the hidden ARIA live region should be updated with the name of the page.

https://github.com/digitallinguistics/app/blob/318207f387d97911dc60b2216b9e35d3747e39f3/src/index.html#L117-L118
